layers/+tools/tmux/config.vim

```diff
@@ -16,6 +16,8 @@ endif
 
 " fix Ctrl arrow  tmux and vim --> https://stackoverflow.com/questions/15445481/mapping-arrow-keys-when-running-tmux
 if &term =~ '^screen'
+    " make mouse can click to place cursor
+    set mouse=a
```

Comment:
It has been set by default in [vim-better-default](https://github.com/liuchengxu/vim-better-default):

```vim
:verbose set mouse
  mouse=a
        Last set from ~/.vim/plugged/vim-better-default/plugin/default.vim
```